scripts/migrations/v1-add-steady-type-down.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# uses jq, can be installed by npm i -g node-jq
+
+echo "creating backup users.backup.json"
+cp apps/backend/data/users.json apps/backend/data/users.backup.down.json
+
+jq 'del(.[].workouts[].parts[].type)' apps/backend/data/users.backup.down.json > apps/backend/data/users.json
+
+echo "Done. Remember to check result and delete backup if OK :))"
+echo "Result:"
+echo $(cat apps/backend/data/users.json)
```

Comment:
Do we really want/need to print the result? If we want it printed, we might want to consider moving the first print after the result print
```suggestion
echo "Result:"
echo $(cat apps/backend/data/users.json)
echo "Done. Remember to check result and delete backup if OK :))"
```

Reply:
Fixed :))